openmls/src/framing/validation.rs

```diff
@@ -299,30 +299,31 @@ pub struct VerifiedMemberMessage {
 
 impl VerifiedMemberMessage {
     /// Returns a reference to the inner [MlsPlaintext].
-    pub fn plaintext(&self) -> &MlsPlaintext {
+    pub(crate) fn plaintext(&self) -> &MlsPlaintext {
         &self.plaintext
     }
 
     /// Consumes the message and returns the inner [MlsPlaintext].
-    pub fn take_plaintext(self) -> MlsPlaintext {
+    pub(crate) fn take_plaintext(self) -> MlsPlaintext {
         self.plaintext
     }
 }
 
 /// External message, where all semantic checks on the framing have been successfully performed.
+/// Note: External messages are not fully supported yet #106
 pub struct VerifiedExternalMessage {
-    plaintext: MlsPlaintext,
+    _plaintext: MlsPlaintext,
```

Comment:
Is this to avoid unused code warning?
I'd prefer warnings rather than hiding them so we know there's something to fix.

Reply:
That's why I added the note above. I don't think it's a good idea to keep clippy warnings around for a long time (#106 is not part of milestone v0.4), it leads to warning fatigue and we might miss actually important warnings.